README.md

```diff
@@ -128,7 +128,14 @@ Pwwka::Transmitter.send_message!(
 
   * `:raise` - Log the error and raise the exception received from Bunny. (default strategy)
   * `:ignore` - Log the error and return false.
-  * `:resque` - Log the error and return false. Also, enqueue a job with Resque to send the message. See `send_message_async` below. **Note, this doesn't guarantee the message will actually be sent—it just guarantees an attempt is made to queue a Resque job [which could fail]**
+  * `:resque` - Log the error and return false. Also, enqueue a job with Resque to send the message. See `send_message_async` below. **Note, this doesn't guarantee the message will actually be sent—it just guarantees an attempt is made to queue a Resque job [which could fail]**. This is a legacy option and `:retry_async` should be used instead.
```

Comment:
I'd be in favor of removing this options from the README altogether so it only has guidance for the current api